.github/main.workflow

```diff
@@ -0,0 +1,16 @@
+workflow "Build & Release Manifest" {
+  on = "push"
```

Comment:
Do we want to limit this just to master? Is there any benefit of build the manifest on anything else?

Reply:
I don't think the actions definitions have any way to limit by branch. We could limit to master in the action itself, but it's not totally straightforward. Happy to look at that in a followup.

Reply:
Yeah, seemingly not. :(